spec/lib/telephony/otp_sender_spec.rb

```diff
@@ -139,13 +139,18 @@
             <prosody rate='slow'>
               Hello! Your #{APP_NAME} one time passcode is,
               1 <break time='0.5s' /> 2 <break time='0.5s' /> 3 <break time='0.5s' />
-              4 <break time='0.5s' /> 5 <break time='0.5s' /> 6,
-              again, your passcode is,
+              4 <break time='0.5s' /> 5 <break time='0.5s' /> 6.
+
+              Your #{APP_NAME} one time passcode is,
               1 <break time='0.5s' /> 2 <break time='0.5s' /> 3 <break time='0.5s' />
-              4 <break time='0.5s' /> 5 <break time='0.5s' /> 6,
+              4 <break time='0.5s' /> 5 <break time='0.5s' /> 6.
+
+              Again, your passcode is,
+              1 <break time='0.5s' /> 2 <break time='0.5s' /> 3 <break time='0.5s' />
+              4 <break time='0.5s' /> 5 <break time='0.5s' /> 6.
               This code expires in 5 minutes.
             </prosody>
-          </speak>
+        </speak>
```

Comment:
I think this needs the indentation still
```suggestion
          </speak>
```

Reply:
- Maybe another option would be to have like a `start_with` matcher for the message instead of exact equality?
- Also looks like there's another XML blob in the spec below we'll need to fix

Reply:
> * Maybe another option would be to have like a `start_with` matcher for the message instead of exact equality?
> * Also looks like there's another XML blob in the spec below we'll need to fix

The tricky thing with using `start_with?` that I am finding is that it returns true with the value of `<speak>`, not the text itself 🤔 

Also, I am very perplexed by the fact that the authentication block works, but the authentication _and_ confirmation block at the same time does not work.